test/deploy/validate-configs.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+# Script to validate the packaged module
+
+TEST_NAME="Deployment testing"
+mkdir logs
+
+echo "$TEST_NAME - Building and installing package"
+npm pack --loglevel warn
+npm install -g solid-community-server-*.tgz --loglevel warn
+rm solid-community-server-*.tgz
+
+run_server_with_config () {
```

Comment:
Can we remove `validate-package.sh` and its corresponding CI check since this script does the same and more?

Reply:
Should there be a script shortcut added for this test in package-lock.json as well then? Or just replace `npm run test:deploy` with this script?

Reply:
I'd replace indeed. Or maybe this script can optionally take argument(s) to indicate which files to process.

Reply:
That is certainly possible, might come in handy.
So either just `npm run test:deploy`, no arguments checks all configs, if argument is provided will test given config

Or we make two entries:
`npm run test:deploy` would be the version that takes the argument of a config file, no argument runs 'default.json'
`npm run test:deploy-configs` or something similar would then be all configs

Reply:
One entry works for me. Then we can still `npm run test:deploy -- tests`.

Reply:
Might be best in that case to split things up more, since in this script we do a `npm pack` followed by an `npm install`. Which would be overkill if you just want to validate config locally (just build and run in that case, which seems quicker)

Reply:
We could—although it's really a deploy test so it's doing the right thing for me.